docs/cellxgene_census_schema.md

```diff
@@ -87,132 +87,172 @@ For a given multi-species dataset, the table below shows all possible combinatio
-    <td></td>
+  </tr>
+  <tr>
+    <td>EFO:0700011</td>
+    <td>GEXSCOPE technology</td>
   </tr>
   <tr>
     <td>EFO:0008780</td>
     <td>inDrop</td>
-    <td></td>
   </tr>
   <tr>
-    <td>EFO:0008913</td>
-    <td>single-cell RNA sequencing</td>
-    <td>Refers to "TruDrop" from this <a href="https://cellxgene.cziscience.com/collections/a48f5033-3438-4550-8574-cdff3263fdfd">collection</a> which is not included in EFO at the moment.</td>
+    <td>EFO:0008796</td>
+    <td>MARS-seq</td>
   </tr>
   <tr>
-    <td>EFO:0008919</td>
-    <td>Seq-Well</td>
-    <td></td>
+    <td>EFO:0030002</td>
+    <td>microwell-seq</td>
   </tr>
   <tr>
-    <td>EFO:0008930</td>
-    <td>Smart-seq</td>
-    <td></td>
+    <td>EFO:0010550</td>
+    <td>sci-RNA-seq</td>
   </tr>
   <tr>
-    <td>EFO:0008931</td>
-    <td>Smart-seq2</td>
-    <td></td>
+    <td>EFO:0008919</td>
+    <td>Seq-Well</td>
   </tr>
   <tr>
-    <td>EFO:0008953</td>
-    <td>STRT-seq</td>
-    <td></td>
+    <td>EFO:0030019</td>
+    <td>Seq-Well S3</td>
   </tr>
   <tr>
-    <td>EFO:0008995</td>
-    <td>10x technology</td>
-    <td></td>
+    <td>EFO:0008930</td>
+    <td>Smart-seq</td>
   </tr>
   <tr>
-    <td>EFO:0009899</td>
-    <td>10x 3' v2</td>
-    <td></td>
+    <td>EFO:0700016</td>
+    <td>Smart-seq v4</td>
   </tr>
   <tr>
-    <td>EFO:0009900</td>
-    <td>10x 5' v2</td>
-    <td></td>
+    <td>EFO:0008931</td>
+    <td>Smart-seq2</td>
   </tr>
   <tr>
-    <td>EFO:0009901</td>
-    <td>10x 3' v1</td>
-    <td></td>
+    <td>EFO:0008953</td>
+    <td>STRT-seq</td>
   </tr>
   <tr>
-    <td>EFO:0009922</td>
-    <td>10x 3' v3</td>
-    <td></td>
+    <td>EFO:0700010</td>
+    <td>TruDrop</td>
   </tr>
+</tbody>
+</table>
+
+The Census MUST *exclude* cells with anny `assay_ontology_term_id` value not present in the above list. The rational for excluding assays includes:
+
+<table>
+<thead>
   <tr>
-    <td>EFO:0010010</td>
-    <td>CEL-seq2</td>
-    <td></td>
+    <th>ID</th>
+    <th>Label</th>
+    <th>Reason for exclusion</th>
   </tr>
+</thead>
+<tbody>
   <tr>
-    <td>EFO:0010183</td>
-    <td>single cell library construction</td>
-    <td>Refers to "GEXSCOPE" from this <a href="https://cellxgene.cziscience.com/collections/edb893ee-4066-4128-9aec-5eb2b03f8287">collection</a> which is not included in EFO at the moment.</td>
+    <td>EFO:0030059</td>
+    <td>10x multiome</td>
+    <td>Non-RNA data is not supported</td>
   </tr>
   <tr>
-    <td>EFO:0010550</td>
-    <td>sci-RNA-seq</td>
-    <td></td>
+    <td>EFO:0030007</td>
+    <td>10x scATAC-seq</td>
+    <td>Non-RNA data is not supported</td>
   </tr>
   <tr>
-    <td>EFO:0011025</td>
-    <td>10x 5' v1</td>
-    <td></td>
+    <td>EFO:0008992</td>
+    <td>MERFISH</td>
+    <td>Spatial data is not supported</td>
   </tr>
   <tr>
-    <td>EFO:0030002</td>
-    <td>microwell-seq</td>
-    <td></td>
+    <td>EFO:0008853</td>
+    <td>Patch-seq</td>
+    <td>Non-RNA data is not supported</td>
```

Comment:
Similar to multiome. RNA measurements are included in the assay and that's what we have in the current data.

Reply:
~and~ as it cannot ...

Reply:
LGTM